### PR TITLE
Fix the subtitle interface

### PR DESCRIPTION
**Pull request by SF-AD**
下面就由我来接力吧(

### CODE REVIEW

**Comment by BraveSail** [comment]
https://user-images.githubusercontent.com/49320262/188881049-9ff5279c-5813-4f9f-bd0d-ab58f9621a64.mp4

**Comment by ipcjs** [comment]
加下版本号

File: scripts/bilibili_bangumi_area_limit_hack.user.js [hunk 1]
```diff
@@ -3088,7 +3088,7 @@ function scriptSource(invokeBy) {
                                                             lan: target,
                                                             lan_doc: targetDoc,
                                                             is_lock: false,
-                                                            subtitle_url: `//www.kofua.top/bsub/${converter}?sub_url=${encSubUrl}&sub_id=${encSubId}`,
+                                                            subtitle_url: `//video1.beijcloud.com/sub/t2cn/?sub_url=${origSubUrl}&sub_id=${encSubId}`,
```
Comment (ipcjs) [inline scripts/bilibili_bangumi_area_limit_hack.user.js:3091]: 仔细一看，你好像改错文件了😅
url在这里改：https://github.com/ipcjs/bilibili-helper/blob/user.js/packages/unblock-area-limit/src/main.js#L170

File: scripts/bilibili_bangumi_area_limit_hack.user.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // ==UserScript==
 // @name         解除B站区域限制
 // @namespace    http://tampermonkey.net/
-// @version      8.2.27
+// @version      8.2.28
```
Comment (ipcjs) [inline scripts/bilibili_bangumi_area_limit_hack.user.js:4]: 版本号在这里改：https://github.com/ipcjs/bilibili-helper/blob/user.js/packages/unblock-area-limit/src/main.user.js#L4

**Pull request closed without merging**